7-compute-engine/config/database.example.yml

```diff
@@ -1,5 +1,5 @@
 default: &default
-  dataset_id: @@PROJECT@@
+  dataset_id: PROJECT_ID
```

Comment:
I'm not understanding why we changed from the `@@PLACEHOLDER@@` syntax, nor when we choose to use `PLACEHOLDER` without brackets vs `[PLACEHOLDER]` with brackets.

* If we're concerned about the `@@` characters breaking yaml syntax, then just put quotes around it.
* We're doing some `sed` substitutions in both this tutorial (`startup-script.sh`) and the kubernetes tutorial (`setup.sh`) but we're using different syntaxes between the two. Could we standardize on one syntax?

Reply:
There exist two sections in the compute engine tutorial. One section is for local development and has the user modifying the database.example.yml directly instead of using startup-script.sh. The second section is using `sed` in the startup-scrip.sh. 

They both have placeholders, and I didn't want to confuse users when modifying the startup-script.sh.

I'm open to interpretation, and FWIW, `[placholder]` is a commonly used placeholder format.

Reply:
Hmm. I see. So compute engine is unique in that we want to avoid pushing the customized `settings.yml` into source control, so we get around that by having the user make _the same_ edits to a `startup-script.sh`. And we want to make it clear in in that script what the editable placeholders are. So that explains why this has a second placeholder format (`@@PLACEHOLDER@@`).

But there still remains the question of what's wrong with the status quo? You need two placeholder formats, one for manual editing of the `*.example.yml` files (for local development), and a second for manual editing of the `startup-script.sh`. Currently, we are using `@@PLACEHOLDER@@` for the former, and `[PLACEHOLDER]` for the latter. That sounds perfectly reasonable. Why not keep it as is? Changing `@@PLACEHOLDER@@` to simply `PLACEHOLDER` makes it less clear that it is a placeholder.